+adi/ADIS16480.m

```diff
@@ -38,25 +38,37 @@
         end
     end   
     
-    methods (Access=protected)
-        function numOut = getNumOutputsImpl(~)
-            numOut = 4;
-        end       
-    end
+%     methods (Access=protected)%REMOVED WHILE stepImpl USES varargout
```

Comment:
Addressed this [error](https://github.com/analogdevicesinc/SensorToolbox/pull/10#discussion_r748945441) by removing getNumOutputsImpl definition here. stepImpl and read method now return variable number of output depending on number of enabled channels.

Reply:
That works. Please remove the comments code :)